tests/integration/test_charm.py

```diff
@@ -58,319 +55,353 @@ async def test_build_and_deploy(ops_test: OpsTest):
     assert ops_test.model.applications[DATA_INTEGRATOR].status == "blocked"
 
 
-@pytest.mark.skip  # skipping as we can't deploy MYSQL (https://github.com/canonical/mysql-operator/pull/73)
 async def test_deploy_and_relate_mysql(ops_test: OpsTest):
     """Test the relation with MySQL and database accessibility."""
     await asyncio.gather(
         ops_test.model.deploy(
-            "mysql", channel="edge", application_name=MYSQL, num_units=1, series="focal"
+            MYSQL[ops_test.cloud_name],
```

Comment:
**suggestion** 

for readability, maybe it is best to define a variable at the top of the function 

`charm_data = MYSQL[ops_test.cloud_name]`

which is referenced throughout in this function

This may apply also to the other functions for other technologies. It is a minor point